### PR TITLE
Add new little arc command

### PR DESCRIPTION
**Pull request by AiroPi**
## Description

#4921 
Add a new command to open little arc.
Don't know if it is possible to add an argument with an URL ? Currently it will display the default page, same as a new tab.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ x I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi @AiroPi 👋 

I updated the code a bit, now you can pass an URL as an argument, if not, use the preference

Could you give it a quick spin 😊

**Comment by AiroPi** [comment]
This argument works great, thanks!
But, if the url provided is something like "google.com" instead of "https://google.com/" it throw an error.
I think it is not an important problem though, and it could be improved in the futur.

But I have a question, by throwing an error, the error message appear in the console while being in debug mode. How will it act if used in "production" ? Will the error appear in the raycast window as with showHUD, or need I replace this error throw with showHUD ?

Also I'm new to the "contribution world of open source", so I don't know for who is the "contributors" key in package.json
Should I add myself or not?

Finally, I think an optional argument could be add to other commands such a "Open New Tab" and "Open New Window". Should I open a 2nd PR for this purpose or is it ok if I add them in this PR, while it is not the main goal of this PR ?

Thanks 😄

**Comment by pernielsentikaer** [comment]
Could be something like this 😊

File: extensions/arc/package.json [hunk 1]
```diff
@@ -33,6 +25,31 @@
...
+          "description": "The URL to open in the new tab",
+          "type": "textfield",
+          "required": false,
+          "default": "https://google.com"
+        }
+      ],
+      "arguments": [
+        {
+          "name": "url",
+          "placeholder": "Optional URL",
```
Comment (thomaspaulmann) [inline extensions/arc/package.json:47]:
```suggestion
          "placeholder": "URL",
```

File: extensions/arc/src/types.ts [hunk 2]
```diff
@@ -38,3 +38,7 @@ export type Suggestion = {
   query: string;
   url: string;
 };
+
+export type Arcguments = {
```
Comment (thomaspaulmann) [inline extensions/arc/src/types.ts:42]: Probably best to call this something like `NewLittleArcArguments` to make it more explicit but can be done with the next codechange.

**Comment by thomaspaulmann** [comment]
I think we're good here, aren't we? This looks great. Thanks for your contribution @AiroPi - I leave the merging to @pernielsentikaer given that he pushed some little adjustments.

**Pull request merged**